requirements.txt

```diff
@@ -33,12 +33,14 @@ sentencepiece>=0.2.0
 tokenizers>=0.11.1
 toml>=0.10.2
 # Intel Gaudi 1.16.1 has torch 2.2.0a0+git8964477 with oneMKL
-torch>=2.3.0a0,<3.0.0 ; python_version == '3.10'
+torch>=2.2.2a0,<3.0.0 ; python_version == '3.10'
 torch>=2.3.0,<3.0.0 ; python_version != '3.10'
 tqdm>=4.66.2
-# 'optimum' for Intel Gaudi needs transformers <4.39.0,>=4.38.0
-transformers>=4.41.2
+# 'optimum' for Intel Gaudi needs transformers <4.41.0,>=4.40.0
+transformers>=4.40.0; python_version != '3.10'
+transformers>=4.41.2 ; python_version != '3.10'
```

Comment:
Should one of these be `==`?

Reply:
Thanks, I fixed this problem and the dup `trl` problem.